pkg/targets/adapter/datadogtarget/config.go

```diff
@@ -30,6 +30,9 @@ type envAccessor struct {
 	// APIKey defines a Datadog API key to be used for authentication
 	APIKey string `envconfig:"DD_CLIENT_API_KEY" required:"true"`
 
+	// Site defines the site of the Datadog intake API, defaults to `datadoghq.com`
+	Site string `envconfig:"DD_CLIENT_SITE" default:"datadoghq.eu"`
```

Comment:
We should default it to the previous value `datadoghq.com` to make it backsward compatible (also the comment points to the previous value so most probably this is a glitch)